### PR TITLE
[plugin.video.manoto] 2.0.0

### PR DESCRIPTION
**Pull request by bbk79**
### Description
Fixing the broken plugin, none of the existing logic is required anymore and a direct link to the m3u8 playlist will work.
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.manoto/default.py [hunk 1]
```diff
@@ -1,115 +1,10 @@
-import urllib,urllib2,re,os,cookielib,string
-import xbmcplugin,xbmcgui,xbmcaddon
-from BeautifulSoup import BeautifulSoup
-
-addon = xbmcaddon.Addon('plugin.video.manoto')
-profile = xbmc.translatePath(addon.getAddonInfo('profile'))
-
+import xbmc, xbmcgui, os, xbmcaddon
 
 __settings__ = xbmcaddon.Addon(id='plugin.video.manoto')
```
Comment (enen92) [inline plugin.video.manoto/default.py]: `id` is not needed. `xbmcaddon.Addon()` is enough

File: plugin.video.manoto/default.py [hunk 2]
```diff
@@ -1,115 +1,10 @@
...
-__language__ = __settings__.getLocalizedString
-
 home = __settings__.getAddonInfo('path')
 icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('manoto')
+li.setThumbnailImage(icon)
```
Comment (enen92) [inline plugin.video.manoto/default.py]: `setThumbnailImage` is deprecated, please use `setArt` instead. See: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html

**Comment by bbk79** [comment]
Updated according to feedback

File: plugin.video.manoto/default.py [hunk 3]
```diff
@@ -1,115 +1,10 @@
...
-
-__settings__ = xbmcaddon.Addon(id='plugin.video.manoto')
-__language__ = __settings__.getLocalizedString
+import xbmc, xbmcgui, os, xbmcaddon
 
+__settings__ = xbmcaddon.Addon()
 home = __settings__.getAddonInfo('path')
 icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('manoto')
+li.setArt(icon)
```
Comment (enen92) [inline plugin.video.manoto/default.py]: setArt receives a dictionary of values, so it has to be something like `li.setArt({'icon':icon})` . Check the documentation for all the options setArt provides.

Comment (bbk79) [inline plugin.video.manoto/default.py]: Oops, sorry about that! updated :)

**Pull request merged**